tests/data/factors-response.json

```diff
@@ -204,5 +204,43 @@
       "input_pointer": "/account/username_md5",
       "warning": "Encountered value at /account/username_md5 that does meet the required constraints"
     }
+  ],
+  "risk_score_reasons": [
+    {
+      "multiplier": 45,
```

Comment:
I wonder if we should use an explicit float in all these examples/fixtures and not just the Java (and maybe .NET) ones